examples/scripts/modbus-thing.jsonld

```diff
@@ -27,7 +27,8 @@
             "writeproperty"
           ],
           "modbus:function": 5,
-          "modbus:range": [0, 1],
+          "modbus:offset": 0,
+          "modbus:length": 1,
           "modbus:unitID": 1
```

Comment:
I think, a good best practice would be that the unitID would be a good place in the `href` or better in the `base` since it provides information about the generic location of the device.

Reply:
I moved everything in href. The base is another good place, but maybe for the example, it is better to explicitly show the URL in the form.